src/containers/genePage/genomeFeatureWrapper.js

```diff
@@ -77,7 +77,8 @@ class GenomeFeatureWrapper extends Component {
   }
 
   getSpeciesString(species) {
-    return SPECIES.find(s => s.fullName === species).apolloName;
+    // short-name is for SARS only
+    return SPECIES.find(s => (s.fullName === species) || (s.shortName === species)).apolloName;
```

Comment:
Doing this lookup by taxon ID is probably better than `fullName` or `shortName`. Not sure if this component has access to that info currently.